### PR TITLE
6824 update import practitioners script, readme and sample CSV

### PR DESCRIPTION
**Pull request by mrinsin**
@mmarcotte to answer some questions from yesterday:
- `name` is computed from the first, middle and last name
- the `inactivePractitioner` role is set based on `admissionStatus` not being 'Active'

### CODE REVIEW

File: docs/IMPORTING_PRACTITIONER_USERS.md [hunk 1]
```diff
@@ -10,3 +10,17 @@ cd web-api
 A sample CSV file is included in the repo: [practitioner_users.csv](../web-api/practitioner_users.csv). The data fields should be in the same order as the provided CSV file.
 
 Running the script will generate a log file, `bulk-import-log.txt`. Users that were successfully created will receive an email with login instructions. Any errors will be printed to the log file for further evaluation.
+
+Notes:
+1. If there is no postalCode available 
+    a. For DOMESTIC country types, postalCode should defaults to '00000' when there is no value supplied
```
Comment (rachaelparris) [inline docs/IMPORTING_PRACTITIONER_USERS.md]: Nit: `should default to`

File: docs/IMPORTING_PRACTITIONER_USERS.md [hunk 2]
```diff
@@ -10,3 +10,17 @@ cd web-api
...
+    b. For INTERNTIONAL country types, postalCode should be set to the country name and defaults to 'N/A' when there is no value supplied
```
Comment (rachaelparris) [inline docs/IMPORTING_PRACTITIONER_USERS.md]: Nit: typo `INTERNATIONAL`

**Review by rachaelparris — APPROVED** [review]

File: docs/IMPORTING_PRACTITIONER_USERS.md (hunk 1 above)
Comment (mrinsin) [reply docs/IMPORTING_PRACTITIONER_USERS.md]: Thanks!

File: docs/IMPORTING_PRACTITIONER_USERS.md (hunk 2 above)
Comment (mrinsin) [reply docs/IMPORTING_PRACTITIONER_USERS.md]: Thanks!

File: shared/src/persistence/dynamo/users/createPractitionerUser.js [hunk 3]
```diff
@@ -52,10 +52,15 @@ exports.createUserRecords = async ({ applicationContext, user, userId }) => {
...
+    ROLES.privatePractitioner,
+    ROLES.irsPractitioner,
+    ROLES.inactivePractitioner,
+  ];
 
-  if (![ROLES.privatePractitioner, ROLES.irsPractitioner].includes(user.role)) {
+  if (!practitionerRoleTypes.includes(user.role)) {
     throw new Error(
-      'Practitioner users must have either private or IRS practitioner role',
+      'Practitioner users must have either private IRS practitioner role, or be an inactive practitioner',
```
Comment (mmarcotte) [inline shared/src/persistence/dynamo/users/createPractitionerUser.js:63]: It's persnickety, but could you make this error include "Private Practitioner"

File: shared/src/persistence/dynamo/users/createPractitionerUser.test.js [hunk 4]
```diff
@@ -193,11 +234,11 @@ describe('createPractitionerUser', () => {
     ).toBeCalled();
   });
 
-  it('should throw an error when attempting to create a user that is not role private or IRS practitioner', async () => {
+  it('should throw an error when attempting to create a user that is not role private, IRS practitioner or inactive practitioner', async () => {
     await expect(
       createPractitionerUser({ applicationContext, user: otherUser }),
     ).rejects.toThrow(
-      'Practitioner users must have either private or IRS practitioner role',
+      'Practitioner users must have either private IRS practitioner role, or be an inactive practitioner',
```
Comment (mmarcotte) [inline shared/src/persistence/dynamo/users/createPractitionerUser.test.js:241]: oop, here, too.

**Pull request merged**